sbpy/units/tests/data/hi05070405_9000036-avg-spec.txt

```diff
@@ -0,0 +1,527 @@
+# MSP Kelley 2019 Apr 16
+#
+# Data from McLaughlin et al 2014.  Deep Impact 9P/Tempel Encounter -
+# Reduced HRII Spectra V3.0, DIF-C-HRII-3/4-9P-ENCOUNTER-V3.0, NASA
+# Planetary Data System.
+#
+# Mid time ~ 2005-07-04 05:32 UT
+#
+# Averaged over illuminated area (29.81 km**2) defined using a pixel threshold
+#
+# J-Y Li 2019 Apr 18
+#
+# Added columns 'flux_sun', 'flux_sun_nu', ref
+# Solar fluxes are calculated with sbpy.spectroscopy.Sun.from_default().observe()
```

Comment:
Note the actual spectrum used in case the default changes in the future.

Reply:
Good point.  Reference added.